Packs/Jira/Integrations/JiraV3/JiraV3_test.py

```diff
@@ -8,12 +7,12 @@
 
 
 def util_load_json(path: str):
-    with io.open(path, mode='r', encoding='utf-8') as f:
+    with open(path, encoding='utf-8') as f:
         return json.loads(f.read())
 
 
 def util_load_bytes_file(path: str):
-    with io.open(path, mode='rb') as f:
+    with open(path, mode='rb') as f:
```

Comment:
the best practice here would be `return Path(path).read_bytes()`, but maybe not in this PR.

Reply:
Can such a suggestion be added using ruff in the pre-commit command?

Reply:
Yes. For this to happen:
1. Add this test to `flake8-use-pathlib` (hope they approve it), let them assign some error code
2. Use the same error code to suggest this change to Ruff